.github/workflows/publish.yml

```diff
@@ -0,0 +1,19 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [published]
```

Comment:
This should also have an option for manual trigger

Reply:
I thought about that, but having a manual trigger would allow project members to publish before merging to main. it would effectively circumvent branch protection